src/qt/transactionview.h

```diff
@@ -69,6 +70,12 @@ class TransactionView : public QWidget
     QLineEdit *addressWidget;
     QLineEdit *amountWidget;
 
+    QTimer *inputDelayPrefix;
```

Comment:
I also like the m_ convention but I don't see it in any of the other Qt code.
Is the plan to slowly change to m_ over time or maybe take the effort to change (part of) the full source code at ones? Two different styles in the same file is not so nice I think.